Octokit.Tests/Reactive/ObservableAssigneesClientTests.cs

```diff
@@ -110,6 +110,68 @@ public void EnsuresNonEmptyArguments()
             }
         }
 
+        public class TheAddAssigneesMethod
+        {
+            [Fact]
+            public void RequestsCorrectUrl()
+            {
+                var newAssignees = new AssigneesUpdate(new List<string>() { "assignee1", "assignee2" });
+                var gitHubClient = Substitute.For<IGitHubClient>();
+                var client = new ObservableAssigneesClient(gitHubClient);
+
+                client.AddAssignees("fake", "repo", 2, newAssignees);
+
+                gitHubClient.Issue.Assignee.Received().AddAssignees("fake", "repo", 2, newAssignees);
+            }
+
+            [Fact]
+            public void EnsuresNonNullArguments()
+            {
+                var githubClient = Substitute.For<IGitHubClient>();
+                var client = new ObservableAssigneesClient(githubClient);
+                var newAssignees = new AssigneesUpdate(new List<string>() { "assignee1", "assignee2" });
+
+                Assert.Throws<ArgumentNullException>(() => client.AddAssignees(null, null, 2, newAssignees));
```

Comment:
you dont need to test combinations of more than one field being null or empty at a time, since the first assert to match, will throw an exception.

In other words if there are 3 input fields that have NotNull asserts, there should be 3 test cases (each one specifying null for 1 field and valid value for the other fields), eg:
null, "name", 2, newAssignee
"owner", null, 2, newAssignee
"owner", "name", 2, null